style.css

```diff
@@ -0,0 +1,1034 @@
+  margin-top: 1rem;
+  font-family: "Lato", sans-serif;
+  color: #69696b;
+  font-size: 11px;
+  margin-left: 48px;
+}
+
+.text-containerAbout-page {
+  background-color: white;
+  text-align: center;
+  font-family: "Lato", sans-serif;
+  line-height: 1.9;
+  margin: 10px;
+  border-width: 1px;
+  border-style: solid;
+  border-color: white;
+}
+
+.text-containerAbout-page p {
+  font-family: "Lato", sans-serif;
+  color: #606161;
+  font-size: 14px;
+  text-align: center;
+  margin-left: 5px;
+  line-height: 1.9;
+}
+
+.dateAbout {
+  color: #4d4e50;
+  font-size: 27px;
+  font-family: sans-serif;
+  margin: 20px;
+  font-weight: 600;
+  margin-left: 6px;
+  margin-bottom: 7vw;
+  line-height: 0.7;
+  text-align: center;
+}
+
+.dateAbout .dateAbout {
+  font-size: 12px;
+  color: #272a31;
+  font-family: "Lato", sans-serif;
+  font-weight: normal;
+}
+
+.dateAbout .locationAbout {
+  -webkit-text-decoration-line: underline;
+  text-decoration-line: underline;
+  color: #4f4f52;
+  font-size: 11px;
+  font-weight: 600;
+  margin-bottom: 10px;
+}
+
+.PastContainer .pastImg.first:hover,
+.PastContainer .partImage.first:active {
+  background-image:
+    url("assets/back-images.jpg"),
+    -webkit-gradient(linear, left top, left bottom, from(rgba(255, 255, 255, 0)), to(rgba(255, 255, 255, 0)));
+  background-image:
+    url("assets/back-images.jpg"),
+    -o-linear-gradient(rgba(255, 255, 255, 0), rgba(255, 255, 255, 0));
+  background-image:
+    url("assets/back-images.jpg"),
+    linear-gradient(rgba(255, 255, 255, 0), rgba(255, 255, 255, 0));
+}
+
+.PastContainer .pastImg.second:hover,
+.PastContainer .pastImg.second:active {
+  background-image:
+    url("assets/back-images1.jpg"),
+    -webkit-gradient(linear, left top, left bottom, from(rgba(255, 255, 255, 0)), to(rgba(255, 255, 255, 0)));
+  background-image:
+    url("assets/back-images1.jpg"),
+    -o-linear-gradient(rgba(255, 255, 255, 0), rgba(255, 255, 255, 0));
+  background-image:
+    url("assets/back-images1.jpg"),
+    linear-gradient(rgba(255, 255, 255, 0), rgba(255, 255, 255, 0));
+}
+
+.PastContainer .pastImg h2:hover,
+.PastContainer .pastImg h2:active {
+  font-size: 30px;
+  color: #ec5242;
+  text-shadow: 10px 10px 10px;
+}
+
+.cancel {
+  -ms-flex-item-align: end;
+  align-self: flex-end;
+  margin-right: 2rem;
+  margin-top: -4rem;
+  width: 19px;
+  height: 18px;
+}
+
+body .hidden {
+  display: none;
+}
+
+.featuredSpeakers .speakers li.hide.hidden {
+  display: none;
+}
+
+.menu.mobile.hidden {
+  display: none;
+}
+
+@media only screen and (min-width: 768px) {
+  body .mobile {
+    display: none;
+  }
+
+  body .Desktop {
+    display: block;
+  }
+
+  .menu.Desktop {
+    display: flex;
+    flex-direction: row;
+    background-color: white;
+    align-items: center;
+    width: 200%;
+    height: 5rem;
+    margin-left: -1rem;
+    box-shadow: 3px 3px 3px hsl(240deg 8% 46%);
+  }
+
+  .menu-Desktop {
+    display: flex;
+    margin: 0;
+    width: 49%;
+  }
```

Comment:
- [ ] [OPTIONAL] I would suggest adding the property `cursor: pointer` to the header links for a better user experience so the users know when they are hovering a link :wink: